src/main/scripts/src/app/workspace/workspace.module.ts

```diff
@@ -34,13 +38,17 @@ import {SharedModule} from '../shared/shared.module';
     WorkspaceRoutingModule
   ],
   declarations: [
-    OrganizationEditComponent,
-    ProjectEditComponent,
+    OrganizationFormComponent,
+    ProjectFormComponent,
+    OrganizationPermissionsComponent,
+    ProjectPermissionsComponent,
+    OrganizationSettingsComponent,
+    ProjectSettingsComponent,
     WorkspaceChooserComponent
   ],
   exports: [
-    OrganizationEditComponent,
-    ProjectEditComponent,
+    OrganizationFormComponent,
+    ProjectFormComponent,
```

Comment:
I do not think these components need to be exported since they are not used in any other modules.